waspc/data/Generator/templates/server/src/_types/index.ts

```diff
@@ -0,0 +1,55 @@
+{{={= =}=}}
```

Comment:
I couldn't get Git to treat this change as a move. Compare with "deleted file" `types/index.ts` to see the true differences.